### ISSUE TITLE
FR: Visualize how the rule will work

### ISSUE THREAD

**Issue by Drugoy** [tumpio/requestcontrol#33]
Feature request:
Example URL and result visualization - this is the most important feature your add-on lacks and needs it badly.
When a user writes a new rule - he can not know if the rule he is writing is correct or not, whether it will match the example URL or not, is the result of the redirection going to be the desired one or not.
How to test all that?

Just take a look at how awesome it was done in Redirector: 
<img src="http://i.imgur.com/nHGQoO7.png">
It makes rule creation pretty much obvious, the regex pattern gets tested on the go as you type it and thus you can see if it matches the example URL and what the output URL is going to look like.

**Comment by tumpio** [comment]
That looks a nice feature from Redirector! I may try to add it to RC when I have time for it.

By the way, are there some problems with Redirector that prevents you from using it?

**Comment by tumpio** [comment]
Any help is appreciated.

**Comment by Drugoy** [comment]
> By the way, are there some problems with Redirector that prevents you from using it?

Yes.
1. It is abandonware.
2. I was quite happy with it until I figured out it got broken and doesn't handle POST requests at all.

I've started looking for better alternatives and only 3 came up, one of which is yours and although it misses some functionality of Redirector - it also has something that Redirector never had (filtering query params, which I requested for Redirector ages ago, but the author replied that he had no time for that project anymore).
